.ci/configuration_aarch64.yaml

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2018 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# test components
```

Comment:
Can you add a comment in this file explaining what it's for please?

Reply:
OK. maybe "Load specific integration units for aarch64"